src/WebJobs.Script/Environment/EnvironmentSettingNames.cs

```diff
@@ -29,6 +29,9 @@ public static class EnvironmentSettingNames
         public const string AppInsightsAuthenticationString = "APPLICATIONINSIGHTS_AUTHENTICATION_STRING";
         public const string AppInsightsEventListenerLogLevel = "APPLICATIONINSIGHTS_EVENT_LISTENER_LOGLEVEL";
         public const string AppInsightsQuickPulseAuthApiKey = "APPINSIGHTS_QUICKPULSEAUTHAPIKEY";
+        public const string OtlpEndpoint = "OTEL_EXPORTER_OTLP_ENDPOINT";
+        public const string OpenTelemetryEventListenerLogLevel = "OTEL_EVENT_LISTENER_LOGLEVEL";
+        public const string TelemetryMode = "AzureFunctionsJobHost:telemetryMode";
```

Comment:
We shouldn't use `:` in env settings, not all platforms allow that. Also, lets choose an existing prefix / naming convention. There are several in this file so we will need to choose one. We also do not use `AzureFunctionsJobHost` anywhere for env variables.

Options:
`FUNCTIONS_HOST_TELEMETRY_MODE`
`AzureWebJobsHostTelemetryMode`

Reply:
No longer need this env variable.